scripts/circle-ci/run-circle-tests.sh

```diff
@@ -13,6 +13,9 @@ function checkDocsBuild {
 CONTAINER_1=(':atlasdb-cassandra-integration-tests:check')
 
 CONTAINER_2=(':atlasdb-ete-tests:check')
+if [[ $INTERNAL_BUILD == true ]]; then
+    CONTAINER_2+=(':timelock-server:dockerTag')
```

Comment:
Rather than adding the task here, we should add `:timelock-server:dockerTag` as a dependency of the relevant ETE tests.